_posts/builtin/2017-10-23-nvidiatm-triton-server-inference-engine.md

```diff
@@ -20,6 +20,7 @@ The parameters used to configure the Triton Service are the following:
  - **Inference Models**: a comma-separated list of inference model names that the server will load. The models have to be already present in the filesystem where the server is running. This option simply tells the server to load the given models from a local or remote repository.
  - **Local backends path**: Only for a local instance, specify the path on the filesystem where the backends are stored.
  - **Optional configuration for the local backends**: Only for local instance, a semi-colon separated list of configuration for the backends. i.e. tensorflow,version=2;tensorflow,allow-soft-placement=false 
+ - **Timeout (in seconds) for time consuming tasks**: Timeout (in seconds) for time consuming tasks like server startup, shutdown or model load. If the task exceeds the timeout the operation will be terminated with an error.
```

Comment:
I think that a comma is needed. But I may be wrong:
If the task exceeds the timeout, the operation will be terminated with an error.